packages/titanium-docgen/validate.js

```diff
@@ -768,15 +784,27 @@ let basePath = '.';
 if (argc > 2) {
 	for (let x = 2; x < argc; x++) {
 		switch (process.argv[x]) {
-			case '--help' :
+			case '--help':
 				cliUsage();
 				process.exit(0);
 				break;
-			case '--standalone' :
-			case '-s' :
+			// TODO: Remove standalone mode? It really just ignores being unable to resolve any types
+			// We should probably just make it like an auto-whitelist for commonly-referred-to types in SDK like Ti.Proxy, Ti.UI.View
+			case '--standalone':
+			case '-s':
 				standaloneFlag = true;
 				common.log('Standalone mode enabled. Errors will not be logged against inherited APIs.');
 				break;
+			case '--whitelisted':
+			case '-w' :
+				// Read next arg as a whitelisted type
+				if (x === argc - 1) {
+					common.log(common.LOG_WARN, 'Must supply name of whitelisted type');
+					cliUsage();
+					process.exit(1);
+				}
+				whitelist.push(process.argv[++x]);
```

Comment:
Should we support a comma separated list of types here? That way one doesn't have to repeat the `-w` flag over and over again.

```suggestion
				whitelist.push(...process.argv[++x].split(','));
```

Reply:
Sounds good to me. This isn't using something nice like commander to process args, so I was relatively cautious in handling it, but this sounds fine.